SwiftUIDZ/SwiftUIDZ/SignUpView.swift

```diff
@@ -0,0 +1,202 @@
+// SignUpView.swift
+// Copyright © RoadMap. All rights reserved.
+
+import SwiftUI
+
+/// Экран входа
+struct SignUpView: View {
+    // MARK: - Constants
+
+    private enum Constants {
+        static let width: CGFloat = 360
```

Comment:
```suggestion
        static let width: CGFloat = 360
```это ширина чего? не понятно